methods.tex

```diff
@@ -75,17 +76,15 @@ \subsection*{Library organization}
 distributions, and more.
 
 \paragraph{\emph{Infrastructure libraries.}} NumPy provides utilities
-for writing unit tests and for building Python packages.
+for writing tests and for building Python packages.
 
 The \code{testing} subpackage provides functions such as
-\code{assert\_allclose(actual, desired)} that may be used in unit
+\code{assert\_allclose(actual, desired)} that may be used in
 test suites for code that uses NumPy arrays.
 
-NumPy provides a subpackage includes functions and classes
-that can be used by libraries that have NumPy as dependency,
-to be used in the library's configuration code that installs the
-library and that builds a package that can be used, for example,
-on the PyPI web site.
+NumPy provides the subpackage \code{distutils} which includes functions and classes
+to facilitate configuration, installation, and packaging of libraries depending on NumPy.
+These can be used, for example, to as part of publishing to the PyPI website.
```

Comment:
I assume we were talking about distutils here? I tried to split things up a bit, not sure it was fully successfull.

Reply:
"to as part" -> ...

Although, I'm not sure our customizations of distutils have much to do with PyPi?